pages/login.js

```diff
@@ -0,0 +1,45 @@
+export default function Login() {
```

Comment:
I would say we don't even need a page. Just `/login, /logout` etc working. I don't think we need a separate page with a button.

Reply:
did you want me to delete the profile page too?